src/flexmock/api.py

```diff
@@ -424,11 +447,14 @@ def mock_method(runtime_self: Any, *kargs: Any, **kwargs: Any) -> Any:
                 for expectation in reversed(FlexmockContainer.flexmock_objects.get(self, []))
                 if expectation.name == name
             ]
-            error_msg = _format_args(name, arguments)
+            error_msg = (
+                f"Arguments for call {name} did not match expectations:\n"
+                f"  Received call:\t{_format_args(name, arguments)}"
+            )
```

Comment:
if could almost be put here, though it might be a bit too much of a stretch
```suggestion
            error_msg = (
                f"Arguments for call {name} did not match expectations:\n"
                f"  Received call:\t{_format_args(name, arguments)}"
                f"{'' if not expectations else "".join(
                   f"\n  Expected call[{index}]:\t{_format_args(name, expectation._args)}"
                   for index, expectation in enumerate(expectations, 1)
                )}"
            )
```

Reply:
I think this one would become too complex. I'd rather keep the current code because it's more readable.